parachain-tracer/src/progress.rs

```diff
@@ -245,6 +245,16 @@ impl Display for DisputesTracker {
 			},
 		}
 
+		if !self.initiators.is_empty() {
+			for (validator_idx, validator_address) in &self.initiators {
+				writeln!(
+					f,
+					"\t\t\t🚀 Validator initiated dispute: {}",
```

Comment:
Not sure what's the best emoji for initiating a dispute, but I don't view it as a positive event :)
```suggestion
					"\t\t\t😠 Validator initiated dispute: {}",
```